client/src/tasks_sidebar.ts

```diff
@@ -162,9 +162,13 @@ class DenoTaskProvider implements TaskProvider {
           if (!sourceUri) {
             continue;
           }
-          const workspaceFolder = (workspace.workspaceFolders ?? []).find((
-            f,
-          ) => sourceUri.startsWith(f.uri.toString()));
+          const workspaceFolder = (workspace.workspaceFolders ?? []).find((f) =>
+            // NOTE: skipEncoding in Uri.toString(), read more at https://github.com/microsoft/vscode/commit/65cb3397673b922c1b6759d145a3a183feb3ee5d
+            sourceUri
+              .toString(true)
```

Comment:
This doesn't take an argument apparently

Reply:
Yeah, sorry, Totally forgot that `sourceUri` is just string actually.